serverless/pages/ingest-your-data-ingest-data-through-integrations-connector-client.mdx

```diff
@@ -82,20 +82,19 @@ You'll need to check the [individual connector documentation](#available-connect
 ## Initial setup in UI
 
 In your project's UI, go to **((es)) → Connectors**.
-Read the high-level instructions under **Create a connector** to make sure you understand the process.
 Follow these steps:
 
 1. Select **Create a connector**.
 2. Choose a third-party service from the list of connector types.
 3. Add a name and optional description to identify the connector.
 4. Copy the `connector_id`, `service_type`, and `elasticsearch.host` values printed to the screen.
 You'll need to update these values in your [`config.yml`](https://github.com/elastic/connectors/blob/main/config.yml) file.
-5. Make a note of your **((es)) endpoint** and **API key** values. You can create a new API key under **Security → API keys**.
-6. Run the connector code using one of the [options](#deploy-self-managed-connector) documented on this page.
+5. Navigate to  **Elasticsearch → Home**, and make a note of your **((es)) endpoint** and **API key** values. You can create a new API key by clicking on **New** in the **API key** section.
+6. Run the connector code either from source or with Docker, following the instructions below.
 
 ## Deploy your self-managed connector
 
-To use connector clients, you must deploy the connector service so your connector can talk to your ((es)) instance.
+To use connector clients, you must deploy the connector service so your connector can talk to your ((es)) instance. 
 The source code is hosted in the `elastic/connectors` repository.
 
 You have two deployment options:
```

Comment:
We should reverse the order here because Docker is the recommended approach. So Docker first, mentioning that it's the recommended path.

Running from source requires cloning the repository and running the code locally. Use this approach if you're actively customizing connectors.